roles/elasticsearch/defaults/main.yml

```diff
@@ -1,2 +1,15 @@
 ---
+default_admin_user: almalinux
+default_admin_user_become: true
 elasticsearch_version: 8
+cluster_name: prolegis-search
+data_path: /var/lib/elasticsearch
+log_path: /var/log/elasticsearch
+http_port: 9200
+transport_port: 9300
+verify_certs: full
+client_authentication: full
+admin_user: elastic
+dev_user: prolegis_dev
+prod_user: prolegis_prod
+staging_user: prolegis_staging
```

Comment:
these should all be elasticsearch prefixed so `elasticsearch_cluster_name` etc.

Also the defaults should be really not project specific, so the defaults shouldn't have prolegis in them. and we'll set these in our host vars.

Reply:
Done